app/src/main/kotlin/com/wire/android/ui/home/conversations/messagedetails/MessageDetailsReactions.kt

```diff
@@ -67,6 +68,7 @@ fun MessageDetailsReactions(
 }
 
 @MultipleThemePreviews
+@Preview(showBackground = true)
```

Comment:
suggestion: this will actually create 3 different previews: two from `@MultipleThemePreviews` and one additional from the `@Preview(showBackground = true)` - if you want to add background to the previews generated by `@MultipleThemePreviews` then just wrap it in a `Box` with specified background color, but first add `WireTheme` as Damian suggested because it can just be enough to have proper background in preview 😄

Reply:
`WireTheme` and removing redundant `@Preview` was enough :)